gateway-messages/src/sp_to_mgs.rs

```diff
@@ -101,6 +101,9 @@ pub enum SpResponse {
     SendHostNmiAck,
     SetIpccKeyLookupValueAck,
     ComponentSetAndPersistActiveSlotAck,
+
+    /// The size of the trailing caboose data
+    CabooseValue(u32),
```

Comment:
I don't think we need the `u32` (and I think you're actually not using it on the MGS side even in this PR).

Reply:
Good point, removed!